apps/snitch_core/lib/core/data/schema/stock/stock_transfer.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Snitch.Data.Schema.StockTransfer do
+  @moduledoc """
+  Model to track inventory transfer in location
+  """
+  use Snitch.Data.Schema
+  use Snitch.Data.Schema.Stock
+
+  @type t :: %__MODULE__{}
+
+  schema "snitch_stock_transfers" do
+    field(:type, :string)
+    field(:reference, :string)
```

Comment:
1. What is `:reference` for? It is not casted in the changeset and can never be set.
2. There is no reference to `StockItem` or `Variant` so what exactly is being transferred? There's also no `quantity`. :thinking: